scripts/third_party_size_check.py

```diff
@@ -21,35 +21,62 @@
 
 import os
 import sys
+import yaml
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000
 
 
-def _check_size_in_dir(dir_path):
+def _get_skip_files_list():
+    """This function returns the list of the files which are skipped when
+    Oppia is deployed to GAE.
+
+    Returns:
+        list. The list of skip_files directories.
```

Comment:
Please change the description to `The list of files which are to be skipped.`

Reply:
Hi @apb7, Done Also the return types are correct i.e `int` and `list` Thanks!